src/sql/workbench/browser/modelComponents/tree.component.ts

```diff
@@ -110,7 +110,7 @@ export default class TreeComponent extends ComponentBase<azdata.TreeProperties>
 				{
 					indentPixels: 10,
 					twistiePixels: 20,
-					ariaLabel: 'Tree Node'
+					ariaLabel: this.ariaLabel
```

Comment:
I'm not sure if we want to keep the original `Tree Node` text, if `ariaLabel` is empty. I don't think we should, but I'm fine either way.

Reply:
I think removing it is fine - I agree it doesn't seem to provide any value as it is right now